internal/core/metadata/rest_device.go

```diff
@@ -320,18 +320,18 @@ func restGetDeviceById(
 	errorHandler errorconcept.ErrorHandler) {
 
 	vars := mux.Vars(r)
-	var did string = vars[ID]
+	var did = vars[ID]
 
 	res, err := dbClient.GetDeviceById(did)
 	if err != nil {
 		errorHandler.HandleOneVariant(w, err, errorconcept.Database.NotFound, errorconcept.Default.BadRequest)
 		return
 	}
 	w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
-	json.NewEncoder(w).Encode(res)
+	_ = json.NewEncoder(w).Encode(res)
 }
 
-//Shouldn't need "rest" in any of these methods. Adding it here for consistency right now.
+// Shouldn't need "rest" in any of these methods. Adding it here for consistency right now.
```

Comment:
Not a proper docblock comment; should start with function name and describe what function is doing.

May occur elsewhere; please fix all instances.

Reply:
It's an unexported function and GoDoc is not generated for unexported functions.

The question then becomes "is this comment worth keeping in the codebase" and I'd say "probably not."

Reply:
Docblocks are useful for private methods to describe their purpose.  As such, I'd still like to see them for all functions and methods.  I agree that this specific comment isn't worth keeping.  Perhaps replace it with a proper docblock comment that describes the function's purpose/usage.

Reply:
I agree in principle, but what is the scope of this ask? Are you asking for this PR to add comments to every function in this file? Every function in every file touched in this PR? What about those packages?

I'm happy to swing at windmills, just tell me when to stop and what you're looking for.